flink-formats/flink-json/src/test/java/org/apache/flink/formats/json/canal/CanalJsonSerDeSchemaTest.java

```diff
@@ -73,10 +76,69 @@ void testFilteringTables() throws Exception {
                                 InternalTypeInfo.of(PHYSICAL_DATA_TYPE.getLogicalType()))
                         .setDatabase("^my.*")
                         .setTable("^prod.*")
+                        .setIgnoreParseErrors(true)
```

Comment:
What's the purpose of enabling ignore-parse-error for this test case?

Reply:
Sorry, this is redundant.